public/app/features/dashboard-scene/scene/layout-responsive-grid/ResponsiveGridLayoutRenderer.tsx

```diff
@@ -0,0 +1,107 @@
+import { css } from '@emotion/css';
+import { ComponentType, type PointerEventHandler, MutableRefObject, useEffect } from 'react';
+
+import { GrafanaTheme2 } from '@grafana/data';
+import { SceneComponentProps } from '@grafana/scenes';
+import { useStyles2 } from '@grafana/ui';
+
+import { DashboardScene } from '../DashboardScene';
+import { closestOfType } from '../layout-manager/utils';
+
+import { ResponsiveGridItemProps } from './ResponsiveGridItemRenderer';
+import { ResponsiveGridLayout, ResponsiveGridLayoutState } from './ResponsiveGridLayout';
+
+export interface DragAndDropProps {
+  handleProps: {
+    onPointerDown: PointerEventHandler<HTMLElement>;
+  };
+
+  containerProps: {
+    ref: MutableRefObject<HTMLElement>;
+  };
+}
+
+export function ResponsiveGridLayoutRenderer({ model }: SceneComponentProps<ResponsiveGridLayout>) {
+  const { children, isHidden } = model.useState();
+  const styles = useStyles2(getStyles, model.state);
+  const dashboard = closestOfType(model, (s) => s instanceof DashboardScene);
+
+  const layoutOrchestrator = dashboard!.state.layoutOrchestrator!;
+  const { activeLayoutItemRef } = layoutOrchestrator.useState();
+  const activeLayoutItem = activeLayoutItemRef?.resolve();
+  const currentLayoutIsActive = children.some((c) => c === activeLayoutItem);
+
+  useEffect(() => {
+    if (model.containerRef.current) {
+      const computedStyles = getComputedStyle(model.containerRef.current);
+      model.columnCount = computedStyles.gridTemplateColumns.split(' ').length;
+      model.rowCount = computedStyles.gridTemplateRows.split(' ').length;
+
+      // when the contents of a scrollable area are changed, most (all?) browsers
+      // seem to automatically adjust the scroll position
+      // this hack keeps the scroll position fixed
+      if (currentLayoutIsActive && model.scrollPos) {
+        model.scrollPos.wrapper?.scrollTo(0, model.scrollPos.scrollTop);
+      }
+    }
+  });
+
+  if (isHidden || !layoutOrchestrator) {
+    return null;
+  }
+
+  return (
+    <div className={styles.container} ref={model.containerRef}>
+      <div
+        style={{
+          gridRow: model.activeGridCell.row,
+          gridColumn: model.activeGridCell.column,
+          display: currentLayoutIsActive && model.activeIndex !== undefined ? 'grid' : 'none',
+        }}
+      ></div>
+      {children.map((item) => {
+        const Component = item.Component as ComponentType<ResponsiveGridItemProps>;
+        // const Wrapper = isLazy ? LazyLoader : 'div';
```

Comment:
We exported LazyLoader from Scenes in a PR some weeks ago. We should be good to go with using it.